data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			``
+			`		Dear <first>,`
+			`	Interested in an author from the datachip, we are. 'Anna Fangvin', her name is. The story about your race's actions against the oppressors, captivate us it did. To converse with her, we wish.`
+			choice
+				`	(Disregard.)`
+					decline
+				`	(Look for Anna Fangvin.)`
+			`A quick look through your encyclopedia reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview and set a course.`
+	on complete
+		conversation
+			`As you prepare to land, you write Anna a short message, offering an interview. In less than 10 minutes you receive a reply, explaining that Anna has accepted your offer and will wait for you in the spaceport.`
+			`	Look for Anna in the spaceport, so you can take her to the Coalition for a little conversation with those Coalition citizens who showed interest in her work.`
+
+mission "Fangvin: Struggling Author 1"
+	name "Take Anna to Coalition Space"
+	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Book Discovered: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a restaurant. A woman then raises her arms, waving towards you. She must be Anna Fangvin, that author the Coalition citizens showed interest in.`
+			choice
+				`	(Approach her.)`
+			`	You approach the woman and sit right next to her. "Hello, are you <first>?`
+			choice
+				`	"Yes, I am. So, what do you want to ask?"`
+			choice
+				`	"What do you write?"`
+					goto "what do you write"
+			label "what do you write"
+			`	"I write science fiction novels. My latest novel was 'Beyond the Hyperdrive', which I released a few weeks ago. It's about a bunch of humans who steal jump drives from the Quarg. Here, have a copy."`
+			`	Anna retrieves a hardcover book from her bag. It has an image of a Jump Drive on it. You take the book.`
+			`	"You'll like it, I promise."`
+			choice
+				`	"How long is the book?`
+					goto "how long is the book"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+					goto "anna hates quarg"
+			label "how long is the book"
+			`	It isn't very long, it should be three hundred pages.`
+			label "anna hates quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried."`
+			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
```

Comment:
This section too needs to be extended a bit, but not as much as the previous line, I feel the childhood event would take precedence in this conversation. Still, would be better to try and give some more stuff on her continuous not very good encounters with the Quarg. Would break up this section too, but again not as much as the previous line. One thing I'll note is that this should start without her speaking, rather narration on how she's kind of recovering from telling story in the previous line (ie takes some breaths, focuses) before continuing with the rest of her experiences with the Quarg.
A minor thing but something I think would make more sense for the visitting story is to have it involve Lagrange and the ring rather than Humanika. I say this because Humanika is an otherwise common planet, if not for being under the Quarg. I think Lagrange's more alien environment, what with it being a station that (presumably, at least) would reflect something akin to the interior of their ringworlds, I think it's add to the fright young Anna had if she were lost in the long, tall hallways of Lagrange instead of on a planet like Humanika, which she might be more used to by default of being a planet that's pretty habitable for humans. It's also noted the Quarg don't go down there much as the environment being good for humans means it's not good for the Quarg, so I think it needs to be somewhere other than Humanika for the "lost somewhere with no humans in sight, only Quarg" feel, and Lagrange would be the most appropriate option for that as I see it.